### ISSUE TITLE
NTR, pulmo: fibrous alveolar septal thickening

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6394]
**Preferred term label:** 
fibrous alveolar septal thickening

**Synonyms** 
thickened alveolar septa

**Definition (free text, please give PubMed ID)**
thickening of alveolar septa with proliferation of fibrous tissue and inflammatory cells.

ISBN 978-1-4511-7643-8 (alk. paper)
Churg, Andrew, and Churg. Atlas of Interstitial Lung Disease Pathology : Pathology with High Resolution CT Correlations, Wolters Kluwer, 2013. 

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary alveolar system morphology HP:0033137
Pulmonary interstitial high-resolution computed tomography abnormality HP:0025389

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
DIP ORPHA:98852, LIP ORPHA:79128

**Your nano-attribution (ORCID)**
LMU:chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
Given we have the term 
Alveolar septal thickening
Increased width of the alveolar septum, which is the structure that separates neighboring alveoli. This finding can be appreciated on histology.

-- I am wondering if we need this term? It seems that the thickening generally involves some fibrosis and some inflamation, so it is not clear that the subterm is adding new information.
-- It would only make sense to add this subterm if we add at least one other subterm <X> alveolar septal thickening, where X is an adjective that is not ``fibrous``.
@chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
no need for special "fibrous" term

**Comment by pnrobinson** [comment]
Closing, the newly added term ``Alveolar septal thickening`` is sufficient
